doc/nrf/app_dev/build_and_config_system/index.rst

```diff
@@ -165,7 +165,7 @@ MCUboot output build files
 +------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | File                                                             | Description                                                                                                                                                                                                                                                        |
 +==================================================================+====================================================================================================================================================================================================================================================================+
-| :file:`dfu_application.zip`                                      | Contains both the MCUboot-compatible update image for one or more cores and a manifest describing its contents (:file:`app_update.bin` and a :file:`manifest.json` file).                                                                                          |
+| :file:`dfu_application.zip`                                      | Contains both the MCUboot-compatible update image for one or more cores (or slots when MCUboot is in the Direct-XIP mode) and a manifest describing its contents (:file:`app_update.bin` and a :file:`manifest.json` file).                                        |
```

Comment:
Maybe we could inform that there is `manifest.json` and related `*.bin` files. You do not mention all of the files, so maybe such generic information would be better.